OpenRA.Mods.Common/Traits/Render/WithIdleOverlay.cs

```diff
@@ -19,6 +19,9 @@ namespace OpenRA.Mods.Common.Traits
 	[Desc("Renders a decorative animation on units and buildings.")]
 	public class WithIdleOverlayInfo : UpgradableTraitInfo, IRenderActorPreviewSpritesInfo, Requires<RenderSpritesInfo>, Requires<IBodyOrientationInfo>
 	{
+		[Desc("Animation to play when the actor is created.")]
```

Comment:
> Isn't this redundant to WithMakeAnimation.cs?

No, WithMakeAnimation only works with RenderBuilding, and in this case it's the overlay that has a StartSequence, not the building 'body'.